privaterelay/management/commands/invitebetauser.py

```diff
@@ -0,0 +1,18 @@
+from django.core.management.base import BaseCommand
+from django.core.validators import validate_email
+
+from ...models import Invitations
+
+
+class Command(BaseCommand):
+    help = 'Adds an email address to the invitations list/table.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('email', nargs=1, type=str)
```

Comment:
I started the code with the intention to make the user confirm the email address they wanted to add, to prevent any typos from inviting the wrong address. but I didn't end up adding that check anyway, so  no other particular reason except the first use of the command is to invite specific email addresses for some usertesting.com sessions.

Reply:
I see, then do you think we should expand this to invite multiple emails or keep as is?

Reply:
Hmm ... maybe merge now as-is. If we need to add multiple emails in a single command later we can add that.

Reply:
Updated to `invite_beta_user` to match `invite_monitor_waitlist`